### PR TITLE
[RHMIDOC-80] partial_ag - Network Policy

### PR DESCRIPTION
**Pull request by joaedwar**
# Description

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist
- [ ] This change requires a documentation update 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added a test case that will be used to verify my changes 
- [ ] Verified independently on a cluster by reviewer

### CODE REVIEW

**Comment by ldimaggi** [comment]
Question - Is this related to the changes for the VPC/CIDR block?

File: test-cases/tests/documentation/i13-verify-partial-ag-network-policy.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+- Merge Request: https://gitlab.cee.redhat.com/RedHatManagedIntegration-documentation/rhmi-docs/-/merge_requests/133
+- Jira ref link: https://issues.redhat.com/browse/RHMIDOC-80
+- Peer contact: Joan Edwards
+
+* Staging doc link: N/A
+  > **NOTE:** The _Staging doc link_ is not to the Customer Portal. The formatting will not match what is on the Customer Portal; however, the steps should be the same.
+
+## Acceptance Criteria
+
+. Test docs/modules/admin-guide/proc_disabling-communication-between-managed-services-project.adoc
```
Comment (ldimaggi) [inline test-cases/tests/documentation/i13-verify-partial-ag-network-policy.md:18]: The reviewers will need the full URL to access the (correct branch) for these files.

**Comment by joaedwar** [comment]
> Question - Is this related to the changes for the VPC/CIDR block?

This is not related to the VPC/CIDR block; the info on the CIDR values will be in the release notes. I've linked to the related Jira for this doc is in the PR description.

**Pull request merged**